core/port/pool_test.go

```diff
@@ -30,13 +30,13 @@ func TestAcquiredPortsAreUsable(t *testing.T) {
 	pool := NewPool()
 
 	// when
-	tcpPort, _ := pool.Acquire("tcp")
+	tcpPort, _ := pool.Acquire()
 	// then
 	err := listenTcp(tcpPort.Num())
 	assert.NoError(t, err)
 
 	// when
-	udpPort, _ := pool.Acquire("udp")
+	udpPort, _ := pool.Acquire()
```

Comment:
Not sure we need both cases with 'tcp' and with 'udp'. One is enough to test pool.

Reply:
Missed this, thanks.